source/schema-configuration.txt

```diff
@@ -6,8 +6,8 @@ Map Relational Schemas to MongoDB
 
 .. default-domain:: mongodb
 
-Business intelligence tools connect to a data source and, given a fixed
-tabular schema, allow users to visually explore their data.
+Business intelligence tools connect to a data source and, if you provide
```

Comment:
I also think the original sentence was good.
There is really no "if" here. BI tools need a fixed tabular schema.